cfme/tests/configure/test_access_control.py

```diff
@@ -157,6 +158,57 @@ def catalog_obj(appliance):
         cat.delete()
 
 
+@pytest.fixture()
+def tenant_custom_role(appliance, request, provider):
+    """Fixture to create a tenant_administrator role, group, user with additional
+    product features"""
+    # Creating role
+    role = appliance.collections.roles.instantiate(name='EvmRole-tenant_administrator')
+    tenant_role = role.copy(
+        name=fauxfactory.gen_alphanumeric(30, start="EvmRole-tenant_admin_copy_")
+    )
+    with update(tenant_role):
+        if appliance.version < '5.11':
+            tenant_role.product_features = [
+                (['Everything', 'Compute', 'Clouds', 'Auth Key Pairs'], True)
+            ]
+        else:
+            tenant_role.product_features = [
+                (['Everything', 'Compute', 'Clouds', 'Auth Key Pairs'], True)
+            ]
```

Comment:
I don't see any difference between these two values. Also, since 5.10 and earlier versions are no longer being run against master, there's no need for version checking.

Reply:
Done